cbopensource/connectors/yara/bridge.py

```diff
@@ -53,12 +53,23 @@ def analyze_binary(self, md5sum, binary_file_stream):
             raise AnalysisTemporaryError(message="Yara exception", retry_in=10)
         else:
             if matches:
+                score = self.getHighScore(matches)
                 return AnalysisResult(message="Matched yara rules: %s" % ', '.join([match.rule for match in matches]),
                                       extended_message="%s" % ', '.join([match.rule for match in matches]),
-                                      analysis_version=1, score=100)
+                                      analysis_version=1, score=score)
             else:
                 return AnalysisResult(score=0)
 
+    def getHighScore(self, matches):
+        score = 0
+        for match in matches:
+            if match.meta.get('score') > score:
```

Comment:
I couldn't figure a way to use the idiom meta.get('score', 100) in a way that would have preserved a user supplied score over the default score of 100.